packages/ra-core/src/form/getSimpleValidationResolver.spec.ts

```diff
@@ -4,130 +4,221 @@ import {
 } from './getSimpleValidationResolver';
 
 describe('getSimpleValidationResolver', () => {
-    const validator = getSimpleValidationResolver(values => values);
-
-    it('should return a flattened object', async () => {
-        const result = flattenErrors({
-            title: 'title too short',
-            backlinks: [
-                { url: 'url too short', id: 'missing id' },
-                { url: 'url too short', id: 'missing id' },
-            ],
+    describe('flattenErrors', () => {
```

Comment:
dunno :shrug: 

I was able to track its origin down to this issue and especially this comment: https://github.com/marmelab/react-admin/issues/7500#issuecomment-1093085113

But as I said I don't know what made you say this at the time. Since this is currently not documented on rhf's side either I can't tell if there was a change on their side or not...